crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -25,6 +25,9 @@ enum PipelineDescriptor {
     ComputePipelineDescriptor(Box<ComputePipelineDescriptor>),
 }
 
+/// Union of a render and compute pipeline.
```

Comment:
```suggestion
/// A pipeline used to send data to the GPU.
```

The original was redundant, and `union` is a critically distinct keyword in Rust.